.pre-commit-config.yaml

```
@@ -1,6 +1,15 @@
 # Install pre-commit hooks via
 # pre-commit install
 
+default_language_version:
+  python: python3
+
+ci:
+  autofix_prs: true
+  autoupdate_commit_msg: "[pre-commit.ci] pre-commit suggestions"
+  autoupdate_schedule: quarterly
```

```suggestion
  autoupdate_schedule: monthly
```
 to be aligned with dependabot - https://github.com/scientific-python/lazy_loader/blob/2031223913c7e592c1c59aec72f296e9d08171fa/.github/dependabot.yml#L12C18-L12C25